modules/custom/movie_reservation/templates/information-movie.html.twig

```diff
@@ -1,12 +1,29 @@
-<h1> List of movies </h1>
-  <ol>
-    {% for movie in movies %}
-        <li>
-            <div>
-               <h5> {{movie.title.value | raw}}  </h5>
-                    {{movie.field_description.value | raw}}
+{{ attach_library('movie_reservation/movie_reservation') }} 
+<head> 
+    <script src="https://code.jquery.com/jquery-3.6.0.js"></script>
+</head>
+<body>
+    <h1> List of movies </h1>
+    <h5>Please select movie genre for which you would like to make a reservation </h5>
+    <select id="genre" name="genre">
+        <option value="">---Choose a Genre---</option>
+        {% for genre in genres %}
+        <option value="{{genre.tid.value}}"> {{genre.name.value}} </option>
+        {% endfor %}
+    </select> 
+    <ol>
+    {% if movies is not empty  %}
+        {% for movie in movies %}
+            <li>
+                <div>
+                <h5> {{movie.title.value | raw}}  </h5>
+                    {{movie.field_description.value | raw}} 
```

Comment:
Sta predstavlja ovo fileuri dole? Cudno mi je da je u tom formatu naziv.

Reply:
Pronasao sam da nakon updata nove verzije Drupala 8.0.x., da je to najlaksa sintaksa.
https://www.drupal.org/docs/8/api/database-api/dynamic-queries/conditions

Reply:
Kada sam radio prethodni task, probao sam sliku ucitati isto kao i prethodna polja nisam uspeo, pa sam pronasao ovaj deo koda i tako mi je proslo.